.github/workflows/e2e-tests.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install --upgrade pip
-          make install-test-requirements
+          pip install -U .[test]
```

Comment:
Potentially

```suggestion
          uv pip install -U "kedro[test] @ ."
```

Reply:
I tried this, but as I put in the description, there was some weird stuff going on with the installation of `behave` on windows. You can check older builds e.g. https://github.com/kedro-org/kedro/actions/runs/8707116793/job/23881365389